distribute.sh

```diff
@@ -93,6 +93,11 @@ case $OSTYPE in
 		;;
 esac
 
+if [ -f /proc/cpuinfo ]; then
```

Comment:
Didn't think about this situation. I'm not so good in bash just found a variation of this on the web and rewritten it.
Would it be a problem to merge the PR and you could make this tweak?